src/System.Console/ref/System.Console.cs

```diff
@@ -237,6 +247,24 @@ public enum ConsoleKey
         F22 = 0x85,
         F23 = 0x86,
         F24 = 0x87,
+        BrowserBack = 0xA6,
```

Comment:
thanks for adding these!

---
In reply to: [68882217](https://github.com/dotnet/corefx/pull/9741#discussion_r68882217) [](ancestors = 68882217)